pallet/Cargo.toml

```
@@ -0,0 +1,31 @@
+[package]
+name = "beefy-pallet"
```

```suggestion
name = "pallet-beefy"
```
I think that's more in line with other pallets.

---

👍